msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md

```diff
@@ -0,0 +1,139 @@
+
+> [!IMPORTANT]
+> If you have both `TeamsFx` and `TeamsJS` SDK packages in your Teams application,`TeamsFx` SDK is not currently compatible with the latest version (beta version) of `TeamsJS` SDK package. It can cause functional damage to your application when running in Outlook and Office.com.
+> If some functions are not working in Outlook or Office.com, remove `TeamsFx` packages from your application or don't upgrade `TeamsJS` SDK to latest version.
+
+## Prerequisite
+
+Install the latest version of Teams Toolkit from Visual Studio Code extension in [Teams Toolkit (Preview) - Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension).
+
+The following are the prerequisites to upgrade manifest and `TeamsJS` client SDK:
+
+### Upgrade manifest
+
+Run the command `upgrade Teams manifest` to work in Outlook and Office.com and select Teams Tab app manifest file to migrate:
+
+1. `update manifest` version to use the latest `m365DevPreview` schema.
+1. `update manifest` file to use the latest `DevPreview` schema.
+
+To know more about the required manifest schema and version, see [Developer Preview manifest schema](/microsoftteams/platform/resources/schema/manifest-schema-dev-preview)
+
+### Upgrade Teams JavaScript Client SDK
+
+Run the command `Teams: Upgrade Teams JS SDK to extend in Outlook and Office` and select Teams app project to migrate:
+
+1. Update `TeamsJS SDK` to the latest beta version.
+1. Update function references, `Enum` and interface references.
+1. Add `TODO` comments to finish change callbacks manually.
+
+    > [!TIP]
+    > Changing callbacks alters control flow. This code will have `TODO` comments for your review. 
+    > The typical solution is to make parent function `async` and use `await`.
+
+1. Add `TODO` in places where you get changes to the context schema.
+
+## Run your Teams application in Outlook and Office.com
+
+After upgrading you can run Teams application in Outlook and Office.com.
+
+### Upload and run your application in Teams
+
+The following are the steps to Test your application in Outlook and Office.com by uploading your application through Teams client:
+
+1. Push code changes to the server to host your application.
+1. Move application to a zip folder.
+1. Go to Teams client and select **Apps**.
+1. Select **Upload a custom app** and select your application's zip folder.
+1. Select **Add** on app details to install the application.
+
+Teams install and launch your app. You can find your app in **More apps**.
+
+ ![More apps](../assets/images/upgrade-teams/more-apps.png)
+
+## Run Teams application in Outlook
+
+Perform the following steps to preview personal tab apps in Outlook web app and desktop clients:
+
+### Outlook web application
+
+1. Go to https://outlook.office.com 
+1. Select the three dots on the bottom left bar.
+
+    ![More apps view](../assets/images/upgrade-teams/apps.png)
```

Comment:
The steps and the image do not match. Please update images using demo tenant ID and ensure the images are clear and the text content in the images are readable.